addons/onion-skinning/style.css

```diff
@@ -2,13 +2,13 @@
   position: relative;
 }
 .sa-onion-button:focus-within {
-  background-color: hsla(215, 100%, 65%, 0.2);
+  background-color: hsla(260, 60%, 60%, 0.2);
```

Comment:
Editor dark mode and customizable colors doesn't affect this color, but it probably should.
![Purple input background, red buttons](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/7010725f-c1a4-4095-b6b8-86e04927f2e0)